snippets/indexers.mdx

```diff
@@ -30,17 +34,20 @@
 - Norbits
 - OppaiTime
 - Orpheus
-- PornBay
+- PassThePopcorn
+- Piratethenet
+- PixelHD
 - PolishSource
 - PolishTracker
+- PornBay
 - PreToMe
-- PassThePopcorn
```

Comment:
I can see why this got removed but technically we still supported them if they were to fix their IRC and RSS is still a thing so I would leave it in.